docs/Operators.md

```diff
@@ -7135,6 +7136,153 @@ Other versions of this operator: <a href="Changelog.md#If-1">If-1</a>
 </dl>
 
 
+### <a name="ImageToCol"></a><a name="imagetocol">**ImageToCol**</a>
```

Comment:
This op looks to me is a tensor layout change op, which is similar with SpaceToDepth, so putting it into the same folder as "SpaceToDepth" makes more sense.